src/pages/liked-videos-page/LikedVideos.jsx

```diff
@@ -1,7 +1,43 @@
-import React from 'react'
+import React from "react";
+import "./LikedVideos.css";
+import { useContext } from "react";
+import { Aside, VideoCard } from "../../components";
+import axios from "axios";
+import { useEffect } from "react";
+import { LikeVideoContext } from "../../context";
 
-export  function LikedVideos() {
+export function LikedVideos() {
+  const { likeVideos, setLikeVideos } = useContext(LikeVideoContext);
+  const encodedToken = localStorage.getItem("token");
```

Comment:
suggestion - you may extract this line into a function for reuse purposes.

Reply:
can u describe, i didn't get it

Reply:
I meant line 11, so you will just have to run the function whenever you need the token instead of writing the whole logic again.